### PR TITLE
Setting up Prisma. Fixes #5

### LINKED ISSUE #5: Setup Prisma ORM to manage and interact with the database
More info: https://www.prisma.io/express

### PR DESCRIPTION
**Pull request by silva-fj**
Prisma ORM has more features we can benefit from like Type-safe database client and automatically generated SQL database migrations (among other things). It's the most popular ORM for node with first class typescript support nowadays

### CODE REVIEW

File: prisma/migrations/20210707142008_init/migration.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+-- CreateTable
+CREATE TABLE "subscriptions" (
+    "id" SERIAL NOT NULL,
+    "device_token" VARCHAR(255) NOT NULL,
+    "wallet_address" TEXT NOT NULL,
```
Comment (geastwood) [inline prisma/migrations/20210707142008_init/migration.sql:5]: Only wallet address won't cut it. More information related to network is need to uniquely identity. Such as network prefix.

Comment (geastwood) [reply prisma/migrations/20210707142008_init/migration.sql:5]: https://github.com/paritytech/substrate/blob/master/ss58-registry.json

Comment (silva-fj) [reply prisma/migrations/20210707142008_init/migration.sql:5]: I was under the assumption that we will run a service per network, so when connecting an account from the app we would make a request to `/push-service-networkX/subscriptions`. @geastwood What do you think?

**Review by ahoseinian — APPROVED** [review]
LGTM

**Pull request merged**